prboom2/src/g_game.c

```diff
@@ -3179,13 +3182,71 @@ void G_BeginRecording (void)
 {
   int i;
   byte *demostart, *demo_p;
+  int num_requirements = 0;
```

Comment:
The name of this variable is a bit confusing to me. Maybe it should be something like "num_extensions"?